user_docs/en/changes.t2t

```diff
@@ -70,6 +70,7 @@ It is also updated with UIA enabled, when typing text and braille is tethered to
  -
 - Contracted braille input works properly again. (#15773, @aaclause)
 - NVDA is again able to track the focus in applications running within Windows Defender Application Guard (WDAG). (#15164)
+- The Albatross braille driver no longer handles a Esp32 microcontroller as an Albatross display. (#15671)

```

Comment:
In this PR as well as in other ones, you have not added your GitHub's name to the change log item. IMO it would be nice to add it (just my personal feeling).

Adding this credit allows to recognize the work done by contributors, but also allows to know easily and quickly who has been working on which topic, what may be very useful, e.g. for other developers/contributors.
E.g. I know that can ask you something on braille topics, Joseph on UIA or new Windows versions, Luke (XLTechie) on GUI and English documentation, etc.

Adding the GitHub's name is however optional; it remains the choice of each author. So feel free to accept or decline this suggestion:

```suggestion
- The Albatross braille driver no longer handles a Esp32 microcontroller as an Albatross display. (#15671, @burmancomp)
```